e2e/wdio/platform/tests/textarea.e2e-spec.ts

```diff
@@ -138,10 +144,14 @@ describe('Verify Textarea component', () => {
         describe('placeholder should exhibit standard behaviour', () => {
             // The standard behaviour of place holder is under browser control, and can't be checked by webDriver,
             // Checking that placeholder is present.
-            it('should appear as hint text and remain as long as text area is empty', () => {
-                const textAriaPlaceholderBefore = getAttributeByName(basicTextArea, 'placeholder');
-
-                expect(textAriaPlaceholderBefore).toBe(basic_text_area_placeholder);
+            fit('should appear as hint text and remain as long as text area is empty', () => {
```

Comment:
remove `f` from `fit`

Reply:
Let's enhance this https://github.com/SAP/fundamental-ngx/pull/7302